src/CTA.WebForms2Blazor/ControlConverters/GridViewControlConverter.cs

```diff
@@ -21,7 +72,32 @@ public GridViewControlConverter() : base()
 
         public override HtmlNode Convert2Blazor(HtmlNode node)
         {
-            throw new NotImplementedException();
+            node.OwnerDocument.OptionOutputOriginalCase = true;
+            
+            var itemTypeAttr = node.Attributes.AttributesWithName("itemtype").FirstOrDefault();
```

Comment:
Does the `AttributesWithName()` call need to be case-insensitive?

Reply:
After some testing it seems like AttributesWithName() is case insensitive